desktop-environment.el

```diff
@@ -182,6 +192,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % of maximum backlight."
+  (let ((backlight-level (desktop-environment-keyboard-backlight-get)))
+    (if (eq backlight-level 0)
+        (progn (let ((percent "0.0"))
+                 (save-match-data percent)))
+      (progn (let ((percent (*
```

Comment:
the `progn`, `let`, and `save-match-data` are also useless here.

Reply:
See comment above. Removed.